fedora/infra.py

```diff
@@ -134,7 +137,51 @@ async def oncall_remove(self, evt: MessageEvent, username: str) -> None:
         else:
             await evt.reply(f"Unexpected response trying to remove user: {result}")
 
-    @infra.subcommand(name="status", help="get a list of the ongoing and planned outages")
+    # @infra.subcommand(name="status", help="get a list of the ongoing and planned outages")
```

Comment:
Rather than commenting out the code, it's okay to just delete it. If we want it back, it's in git forever. Also, it looks like you got rid of the subcommand decorator which I don't think you intended to do.

Reply:
> Rather than commenting out the code, it's okay to just delete it. If we want it back, it's in git forever. Also, it looks like you got rid of the subcommand decorator which I don't think you intended to do.

Hehe bad habits,sorry. Thank you for pointing that out, no wonder a test i refactored failed